src/Server/ServerAuthorization.luau

```diff
@@ -59,6 +63,13 @@ function ServerAuthorization.new(Configuration: Configuration.Configuration, Nex
     function GetAdminLevels.OnServerInvoke()
         return self.AdminLevels
     end
+    
+    --Initialize flushing GroupAdminLevelCache every 30 minutes.
+    task.spawn(function()
+        while task.wait(30 * 60) do
+            self:FlushGroupAdminLevelCache()
+        end
+    end)
```

Comment:
This cache clearing strategy is simple can lead to an ineffective cache if you are right at the end of the clear period.
Alternative typing I would use for `GroupAdminLevelCache`:
```luau
GroupAdminLevelCache: {[number]: {AdminLevel: number, ExpireTime: number}}
```